src/storage/mod.rs

```diff
@@ -640,6 +641,10 @@ impl<E: Engine, L: LockManager, F: KvFormat> Storage<E, L, F> {
                     .get(priority_tag)
                     .inc();
 
+                if let Err(e) = deadline.check() {
+                    return Err(Error::from(e));
+                }
```

Comment:
```suggestion
                deadline.check()?;
```

And there are some similar usages like this, we can use this syntactic sugar.

Reply:
Great, done.